azure-kusto-data/azure/kusto/data/client_request_properties.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Any
+
+from .kcsb import _assert_value_is_valid
```

Comment:
does this import make sense? seems odd importing a private function from `kcsb`.

Reply:
Fixed